lbrynet/lbrynet_daemon/auth/server.py

```diff
@@ -125,36 +143,35 @@ def expire_session():
 
         try:
             self._run_subhandlers(request)
-        except SubhandlerError:
-            return server.failure
+        except SubhandlerError as err:
+            self._render_error(request, err, version)
+            return server.NOT_DONE_YET
 
         reply_with_next_secret = False
         if self._use_authentication:
             if function_name in self.authorized_functions:
                 try:
                     self._verify_token(session_id, parsed, token)
-                except InvalidAuthenticationToken:
-                    log.warning("API validation failed")
-                    request.setResponseCode(self.UNAUTHORIZED)
-                    request.finish()
+                except InvalidAuthenticationToken as err:
+                    log.error("API validation failed")
```

Comment:
Th InvalidAuthenticationToken error seems like something that would be expected to happen on occasion, so I think the log level here should be at the warning level.
Log statements at the error level should be reserved for things that went wrong and for things we need to fix in the code: https://github.com/lbryio/lbry/wiki/Log-Levels#error.  Logs are not for the user, they are for us.